app/code/Inchoo/Sample08/etc/adminhtml/events.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
+    <event name="controller_action_predispatch">
+        <observer name="adminnotification" disabled="true" />
+    </event>
+
+    <event name="storeCheck" disabled="true"/>
```

Comment:
kaze: Disable storeCheck plugin in adminhtml area.
zašto je disablean event?

Reply:
`storeCheck` je plugin, a ne event (krivi xml)